src/common/test-utils/browser.js

```diff
@@ -5,11 +5,13 @@ const expect = require('chai').expect;
  * @param {HTMLElement} el
  * @param {String} type
  * @param {Number} keyCode
+ * @param {String} key
  */
-function triggerEvent(el, type, keyCode) {
+function triggerEvent(el, type, keyCode, key) {
```

Comment:
I'd move to having an options argument since there are more than 3 arguments. Or have the third argument be an extraEventOptions and then use Object.assign(event, extraEventObject);
That way if we want to add any more items it's a simple object. 
However, if we aren't planning on extending the event object, this could also be fine.

Reply:
I agree an options argument would be better. We should tackle that in a separate PR sometime.

Reply:
👍